docs/examples/auth/basic/README.md

```diff
@@ -40,7 +40,7 @@ metadata:
     # name of the secret that contains the user/password definitions
     nginx.ingress.kubernetes.io/auth-secret: basic-auth
     # message to display with an appropriate context why the authentication is required
-    nginx.ingress.kubernetes.io/auth-realm: "Authentication Required - foo"
+    nginx.ingress.kubernetes.io/auth-realm: Authentication Required - foo
```

Comment:
Indeed, double quotes should be avoided as much as possible in YAML. However I'd prefer to see single quotes here for clarity.